src/utils/Sort.js

```diff
@@ -727,3 +727,16 @@ export function countGroup(group) {
   }
   return group.length;
 }
+
+export function downloadSort(cards) {
```

Comment:
Can you rename this to sortForCSVDownload? downloadSort sounds like it should download a sort.